src/main/java/com/shippingcostestimator/enterprise/service/ShipmentServiceStub.java

```diff
@@ -52,7 +52,7 @@ public Shipment saveEstimate(Shipment shipment) throws Exception {
 
     /***
      * Returns a list of all Shipment objects that are currently in use by the program.
-     * @return
+     * @return a list of all shipments.
```

Comment:
Added return description JavaDoc. [Ref.](https://medium.com/@rhamedy/a-short-summary-of-java-coding-best-practices-31283d0167d3)